features/usage_formatter.feature

```diff
@@ -81,15 +81,15 @@ Feature: usage formatter
     When I run cucumber-js with `--format usage`
     Then it outputs the text:
       """
-      ┌──────────────────┬──────────┬──────────────────────────────────────┐
-      │ Pattern / Text   │ Duration │ Location                             │
-      ├──────────────────┼──────────┼──────────────────────────────────────┤
-      │ /^(slow )?step$/ │ <d>ms    │ features/step_definitions/steps.js:3 │
-      │   slow step      │ <d>ms    │ features/a.feature:3                 │
-      │   slow step      │ <d>ms    │ features/a.feature:3                 │
-      │   slow step      │ <d>ms    │ features/a.feature:3                 │
-      │   slow step      │ <d>ms    │ features/a.feature:3                 │
-      │   slow step      │ <d>ms    │ features/a.feature:3                 │
-      │   5 more         │          │                                      │
-      └──────────────────┴──────────┴──────────────────────────────────────┘
+      ┌──────────────────┬──────────────┬──────────────────────────────────────┐
```

Comment:
I think you are just missing an extra `─` for the `duration` column

Reply:
It  works some times on linux and fails on windows, I'm thinking about a better solution, probably mocking the time on said tests.

Reply:
I just made the normalization function take care of making the number of dashes not matter.
Linux and windows were having different behaviors.